client/arg.cpp

```diff
@@ -159,7 +159,18 @@ bool analyse_argv(const char * const *argv, CompileJob &job, bool icerun, list<s
                 /* as above but with extra argument */
             } else if (!strcmp(a, "-MT") || !strcmp(a, "-MQ")) {
                 args.append(a, Arg_Local);
-                args.append(argv[++i], Arg_Local);
+                ++i;
+                if (argv[i][0] == '"') {
```

Comment:
It turns out that there is no need to handle quotes. I was calling icecc from a shell script that was doing some escaping. I'm reverting the quotes handling patch.